### PR TITLE
add backtrace on panic

### PR DESCRIPTION
**Pull request by Kachulio1**
screen shot 2018-10-31 at 12 40 32

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -66,6 +66,7 @@ fn main() {
   // https://github.com/rust-lang/cargo/issues/2738
   // Therefore this hack.
   std::panic::set_hook(Box::new(|panic_info| {
+    eprintln!("{}", panic_info.to_string());
     if let Some(location) = panic_info.location() {
```
Comment (ry) [inline src/main.rs]: Remove this conditional completely.

**Comment by hayd** [comment]
This is an improvement for sure, but I don't think it solves #1075. Specifically it doesn't say where the offending unwrap was called from, instead it (still) points to libcore/result.rs.

File: src/main.rs (hunk 1 above)
Comment (Kachulio1) [reply src/main.rs]: and keep ```eprintln!("PANIC file '{}' line {}", location.file(), location.line());```

Comment (Kachulio1) [reply src/main.rs]: done

**Pull request merged**